### PR TITLE
Move DiagnosticProperties & DiagnosticUtils to openj9.internal.tools.attach.target

### PR DESCRIPTION
**Pull request by JasonFengJ9**
**Move DiagnosticProperties & DiagnosticUtils to openj9.internal.tools.attach.target**

Moved classes `DiagnosticProperties` & `DiagnosticUtils` into package `openj9.internal.tools.attach.target`;
Updated references accordingly.

Reviewer: @pshipton 
FYI: @DanHeidinga 

Signed-off-by: Jason Feng <fengj@ca.ibm.com>

### CODE REVIEW

File: jcl/src/java.base/share/classes/module-info.java.extra [hunk 1]
```diff
@@ -31,7 +31,7 @@ exports com.ibm.oti.vm to java.management, jdk.attach, jdk.management, openj9.jv
 exports com.ibm.oti.util to java.management, jdk.attach, jdk.jcmd, jdk.management, openj9.sharedclasses;
 exports openj9.internal.tools.attach.target to jdk.attach, jdk.jcmd, jdk.management;
 exports openj9.management.internal to java.management, openj9.jvm;
-exports openj9.internal.tools.attach.diagnostics.base to jdk.attach, jdk.jcmd;
+exports openj9.internal.tools.attach.target to jdk.attach, jdk.jcmd;
```
Comment (pshipton) [inline jcl/src/java.base/share/classes/module-info.java.extra]: This is already done 2 lines above.

**Comment by pshipton** [comment]
jenkins test sanity,extended win,plinux jdk8

**Comment by pshipton** [comment]
https://ci.eclipse.org/openj9/job/PullRequest-OpenJ9/1388/

File: jcl/src/java.base/share/classes/module-info.java.extra (hunk 1 above)
Comment (JasonFengJ9) [reply jcl/src/java.base/share/classes/module-info.java.extra]: Right, removed duplicated line.

**Comment by pshipton** [comment]
jenkins copyright check

**Comment by pshipton** [comment]
jenkins line endings check

**Comment by pshipton** [comment]
@JasonFengJ9 pls check the extended test failures in https://ci.eclipse.org/openj9/job/PullRequest-OpenJ9/1388/

**Comment by JasonFengJ9** [comment]
Looking at it.

**Comment by JasonFengJ9** [comment]
It was `UnsatisfiedLinkError` caused by mismatch natives.
Updating the natives, and run extended tests.

**Comment by pshipton** [comment]
Need to make sure this change is included https://github.com/eclipse/openj9/pull/7309

**Comment by pshipton** [comment]
jenkins test sanity,extended win,plinux jdk8

**Comment by pshipton** [comment]
https://ci.eclipse.org/openj9/job/PullRequest-OpenJ9/1399/

**Comment by pshipton** [comment]
jenkins test sanity,extended win,plinux jdk8

**Comment by pshipton** [comment]
https://ci.eclipse.org/openj9/job/PullRequest-OpenJ9/1401/

**Comment by JasonFengJ9** [comment]
Rebased and included https://github.com/eclipse/openj9/pull/7309.

**Comment by pshipton** [comment]
The PR testing passed.

**Pull request merged**